pipelines/pipeline_katello_upgrade_nightly.yml

```diff
@@ -0,0 +1,85 @@
+- hosts: localhost
+  vars:
+    forklift_name: pipeline-upgrade-nightly
+    forklift_boxes:
+      pipeline-upgrade-centos7:
+        box: centos7
+        memory: 6144
+  roles:
+    - forklift
+
+- hosts: pipeline-upgrade-centos7
+  become: yes
+  vars:
+    puppet_repositories_version: 4
```

Comment:
I'd prefer Puppet 5 unless you upgrade it in the next step.

Reply:
Hmmm, upgrading it might be a good plan for it. I'll change it so that it does that.